app/views/users/show.html.erb

```diff
@@ -5,12 +5,11 @@
     </div>
     <div class="row">
       <div class="col-lg-4">
-        <%= image_tag @user.image.url(:medium) %>
+        <%= image_tag @user.image.url %>
```

Comment:
I'll add image sizes. I want to find out what the sizes need to be.
*But user images are up and running*